src/styles/_colors.scss

```diff
@@ -29,6 +29,26 @@ $tertiary-light-color: lighten($tertiary-color, 10) !default;
 /* Text */
 $text-dark-color: #1C1C1C !default;
 
+/* Damage */
+$damage-physical: #3c3c3c;
+$damage-acid: #007300;
+$damage-bleed: #99001a;
+$damage-bludgeoning: $damage-physical;
+$damage-chaotic: #a600a6;
+$damage-cold: #2f2fa6;
+$damage-electricity: darkgoldenrod;
+$damage-evil: indigo;
+$damage-fire: #a62f00;
+$damage-force: #6300aa;
+$damage-good: #9d730a;
+$damage-lawful: #402600;
+$damage-mental: midnightblue;
+$damage-negative: #00001f;
+$damage-piercing: $damage-physical;
+$damage-poison: #5b7332;
+$damage-positive: #ffffe0;
+$damage-slashing: $damage-physical;
+$damage-sonic: darkcyan;
```

Comment:
Make them CSS variables if you're going to do that

Reply:
I've added CSS variables based on the SCSS variables. Apparently functions like `color.adjust` don't work with CSS variables.

Reply:
Ohhh, right, right.

Reply:
Ehh, let's nix that, then, sorry. All it'll do if anyone tries to override them is not work for some reason.